core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineSqlServer.java

```diff
@@ -78,7 +78,7 @@ public String dropNamespaceSql(String namespace) {
   @Override
   public void dropNamespaceTranslateSQLException(SQLException e, String namespace)
       throws ExecutionException {
-    throw new ExecutionException(String.format("error dropping the schema %s", namespace), e);
+    throw new ExecutionException("dropping the schema failed: %s" + namespace, e);
```

Comment:
Same as above

Reply:
Fixed in 87166ae9c9a4c8f3769d8d53a2d42881ca668d25. Thanks.